gemfirexd/tools/src/test/java/com/pivotal/gemfirexd/internal/engine/FabricServerTest.java

```diff
@@ -675,27 +675,20 @@ public void testNetworkStartupAPI() throws Exception {
 
     System.setProperty(com.pivotal.gemfirexd.Property.PROPERTIES_FILE, PROP_FILE_NAME);
 
-    if (!ClientSharedUtils.isThriftDefault()) {
-      System.setProperty(Property.START_DRDA, "true");
-    }
-    int port;
+    int port, port2;
     while ((port = AvailablePort.getRandomAvailablePort(AvailablePort.SOCKET))
         <= FabricService.NETSERVER_DEFAULT_PORT);
-    if (!ClientSharedUtils.isThriftDefault()) {
-      System.setProperty(com.pivotal.gemfirexd.Property.DRDA_PROP_PORTNUMBER,
-          String.valueOf(port));
-    }
+    while ((port2 = AvailablePort.getRandomAvailablePort(AvailablePort.SOCKET))
+        <= FabricService.NETSERVER_DEFAULT_PORT);
```

Comment:
Yes the probability is very small. But normally in tests don't we use the port first so that isavailable returns false for that and then ask for the next available port.